swift/example-qr-generator/Qr/QrApp.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import pixelpass
+struct QRCodeView: View {
+@State private var inputString = "Test QR Code Generation"
```

Comment:
Can we make the input from user and generate the QR on the fly

Reply:
Done